tests/main/views/test_marketplace.py

```diff
@@ -34,6 +35,10 @@ def test_data_analytics_track_page_view_is_shown_if_account_created_flag_flash_m
         data = res.get_data(as_text=True)
 
         assert 'data-analytics="trackPageView" data-url="buyers?account-created=true"' in data
+        # however this should not be shown as a regular flash message
+        flash_banner_match = re.search('<p class="banner-message">\s*(.*)', data, re.MULTILINE)
```

Comment:
Is there a particular reason we're using `re` rather than `xpath` to look for the element? In the briefs frontend we've been using the latter but I have no idea which is more consistent!